website/content/v1.11/advanced/etcd-maintenance.md

```diff
@@ -74,3 +74,114 @@ NODE         MEMBER             DB SIZE   IN USE             LEADER
 
 Regular backups of `etcd` database should be performed to ensure that the cluster can be restored in case of a failure.
 This procedure is described in the [disaster recovery]({{< relref "disaster-recovery" >}}) guide.
+
+## Downgrade v3.6 to v3.5
+
+### Server downgrade checklists
+
+Before beginning, check the etcd health and download snapshot, as described in [disaster recovery]({{< relref "disaster-recovery" >}}).
+Should something go wrong with the downgrade, it is possible to use this backup to rollback to existing etcd version.
+
+### Downgrade procedure
+
+This example shows how to downgrade an etcd in Talos cluster.
+
+#### Step 1: check downgrade requirements
+
+Is the cluster healthy and running v3.6.x?
+
+```bash
+$ talosctl -n <CP1>,<CP2>,<CP3> etcd status
```

Comment:
Id like to think that CP1, CP2, CP3 are very obviously the IP addresses to the control plane node. But I am not sure if its obvious to everybody else that will come in contact with this docs.

Is it possible to make a slight comment to tell them that those are placeholders and what the placeholders stand for?

Reply:
Added!